test/behaviour/concept/serialization/json/BUILD

```diff
@@ -70,6 +70,7 @@ typedb_behaviour_java_test(
         "//test/behaviour/connection/database:steps",
         "//test/behaviour/connection/session:steps",
         "//test/behaviour/connection/transaction:steps",
+        "//test/behaviour/util:steps"
```

Comment:
Does the JSON step actually use `util:steps`?

Reply:
no, but I thought we might use it in the future, and a good general thing to have. Should I remove it?